### ISSUE TITLE
Error colour fails to meet contrast requirements

### ISSUE THREAD

**Issue by erisan-noroff** [erisan-noroff/compile-javascript-2#48]
# Error Colour Fails to Meet Contrast Requirements

The current error colour #DC2626 does not meet contrast requirements. A new colour that satisfies WCAG 2.1 requirements must be picked. In design mockups, this is used for the Error Toast Notification and form validation errors.

## Acceptance Criteria
- [x] Error colour meets WCAG 2.1 AA for text
- [x] Update design mockups with the new colour
- [x] Updated in DESIGN.md and CSS variables

**Comment by erisan-noroff** [comment]
New colour is `#F85149`. Was picked before the bug was officially created.

**Comment by erisan-noroff** [comment]
The bug fix itself will be worked on in the `23-implement-sign-up-page` branch, as we will implement Toast Notifications and Form Validation there.
